src/features/Dashboard/components/RecentTransactionList.tsx

```diff
@@ -131,11 +131,11 @@ const SCombinedCircle = (asset: Asset) => {
 const makeTxIcon = (type: ITxHistoryType, asset: Asset) => {
   const greyscaleIcon = asset && <>{SCombinedCircle(asset)}</>;
   const baseIcon = (
-    <div className="TransactionLabel-image">
+    <div style={{ marginRight: '16px', position: 'relative' }}>
```

Comment:
Shouldn't we use a styled component for this?

Reply:
IMHO, when the rules fit on the same line, it's better to style in-line rather than create an extra component which you need to name, then scroll back and forth to see which rules are applied.
A style suggestion to apply conventions when useful.

Reply:
Could you use `<Box mr='16px' position='relative'>` for that ?

Reply:
It's only missing the [position](https://styled-system.com/api#position) util

Reply:
That's what I'm suggesting ^